parsec/core/gui/main_window.py

```diff
@@ -680,7 +680,7 @@ def closeEvent(self, event):
         if self.minimize_on_close and not self.need_close:
             self.hide()
             event.ignore()
-            if not self.minimize_on_close_notif_already_send:
+            if not self.minimize_on_close_notif_already_send and platform.system() != "Darwin":
```

Comment:
why the check is done on `sys.platform` in app.py and on `platform.system()` here ?

It seems so far the codebase use `sys.platform` and `platform.system()` interchangeably (and sometime even `os.name`, which is definitely not a good idea given it cannot tell appart Linux from macOS).
From what I understand the difference between the two is `sys.platform` is set at compilation time where `platform.system()` relies on `uname` to return the value (see https://github.com/python/cpython/blob/2f01c562be913004e1d46fc02705c51b81638d67/Lib/platform.py#L1061).

I would say it's better to use `sys.platform` so that we are guaranteed on the returned values (https://docs.python.org/fr/3/library/sys.html#sys.platform)

And I'm going to open an issue about the need to standardize the rest of the codebase ;-)

Reply:
And more comment is needed to explain why macOS do it differently